component-config/secure-api-gateway-ob-uk-functional-tests.env

```diff
@@ -1,4 +1,4 @@
-CF_PIPELINE_BRANCH=master
+CF_PIPELINE_BRANCH=ob-v4
```

Comment:
Will this affect how the tests are run against nightly?

We want nightly to stick to master/latest image and only dev to use the ob-v4/latest-v4 image

Reply:
Good point, focused on core and ob difference, adding in code now to decide on which one to use

Reply:
So if its nightly it will change the value to master, if its dev then it will use the same value as what is in the .env file, so that if we do a v5,6 we only need to change the value in the config file and not the worklows